uint8 bullet_type    # 子弹类型: 1(17mm弹丸), 2(42mm弹丸)
uint8 shooter_id    # 发射机构ID: 1,2为17mm, 3为42mm
uint8 bullet_freq   # 射频(Hz)
float32 bullet_speed # 射速(m/s)
builtin_interfaces/Time stamp # 时间戳 